learninGitHub101.txt

```diff
@@ -1,3 +1,5 @@
 This is Roy, I created this text file on July 17 2017 at 1014 AM pacific standard time.
 
-Please checkout this branch & file, edit it on a new line, save the file up, git commit, and then git push to commit it to the repository.
+Please clone this branch, edit it on a new line, save the file up, git commit, and then git push to commit it to the repository.
```

Comment:
Thx for the email from yesterday, Roy. Will take a look at the Oh Shit Git page and try to wrap my head around this stuff. Have to admit I'm still kind of confused about the whole biz, but other stuff pulling for my attn -- neverending! :-/